### ISSUE TITLE
Fixed VanillaThreadLockTest causing link errors 

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Core#326]
see https://teamcity.chronicle.software/repository/download/OpenHFT_ReleaseJob_ReleaseByArtifact/642376:id/ReleaseAutomation/projects/chronicle-core-runTests-1641549228966.log
 
![image](https://user-images.githubusercontent.com/6506056/148538687-3afa87cf-4912-4a0a-a439-c1931a7e0c63.png)

```
PAUSE1,band,<0.1,<1,<10,<100, 84.5, 100.0, 100.0, 100.0,%iles,50/90 97/99 99.7/99.9 99.97/99.99 - worst was 0.051 / 0.111  0.180 / 0.321  0.401 / 0.401  0.401 / 0.401 - 0.401
BUSY1,band,<0.1,<1,<10,<100, 89.1, 100.0, 100.0, 100.0,%iles,50/90 97/99 99.7/99.9 99.97/99.99 - worst was 0.040 / 0.101  0.131 / 0.192  0.421 / 0.421  0.421 / 0.421 - 0.421
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.17 s - in net.openhft.chronicle.core.io.UnsafeTextTest
[INFO] Tests run: 98, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.045 s - in net.openhft.chronicle.core.UnsafeMemory2Test
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.616 s - in net.openhft.chronicle.core.CleaningRandomAccessFileTest
[WARNING] Tests run: 30, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 11.698 s - in net.openhft.chronicle.core.MathsTest
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   VanillaThreadLockTest.gettid:24 Â» UnsatisfiedLink unknown
[INFO] 
[ERROR] Tests run: 491, Failures: 0, Errors: 1, Skipped: 7
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16.814 s
[INFO] Finished at: 2022-01-07T09:33:04Z
[INFO] ------------------------------------------------------------------------
```

**Comment by peter-lawrey** [comment]
To be deprecated for removal until it is actually used.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Core-2.23ea20](https://github.com/OpenHFT/Chronicle-Core/releases/chronicle-core-2.23ea20), [BOM-2.23ea72](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea72)
